### PR TITLE
`is_shiny_runtime()` is FALSE if knitr is not installed

### PR DESCRIPTION
**Pull request by gadenbuie**
If knitr isn't installed, then `is_shiny_runtime()` is _de facto_ `FALSE`.

This popped up while trying to test using `bs_global_set()` on shinyapps.io where the dependency on knitr may be missed by packrat/renv.

### CODE REVIEW

**Review by schloerke — APPROVED** [review]

File: R/utils.R [hunk 1]
```diff
@@ -44,6 +44,7 @@ is_hosted_app <- function() {
 }
 
 is_shiny_runtime <- function() {
+  if (!is_installed("knitr")) return(FALSE)
```
Comment (cpsievert) [inline R/utils.R:47]:
```suggestion
  if (!is_available("knitr")) return(FALSE)
```

**Review by cpsievert — APPROVED** [review]

File: R/utils.R (hunk 1 above)
Comment (gadenbuie) [reply R/utils.R]: https://github.com/rstudio/bslib/blob/21043820bdbe9b89bd1f9124eb0a91f83614555a/R/staticimports.R#L31

Comment (cpsievert) [reply R/utils.R]: Ack, sorry, will revert change!

File: R/utils.R [hunk 2]
```diff
@@ -44,6 +44,7 @@ is_hosted_app <- function() {
 }
 
 is_shiny_runtime <- function() {
+  if (!is_available("knitr")) return(FALSE)
```
Comment (cpsievert) [inline R/utils.R:47]:
```suggestion
  if (!is_installed("knitr")) return(FALSE)
```

**Pull request merged**